psd-web/app/models/user.rb

```diff
@@ -38,6 +38,10 @@ def self.activated
     where(account_activated: true)
   end
 
+  def self.find_user_in_teams_with!(email, teams)
```

Comment:
Suggestion: rename and use keyword arguments:

```suggestion
  def self.find_user_within_teams_with_email!(teams:, email:)
```